pkg/controller/user-cluster-controller-manager/resources/resources/ca-bundle/configmap.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
why `...resources/resources/ca-bundle`? I mean 2 times of `resources/resources`.

Reply:
Because the controller is badly named ("resources"). It should be `pkg/controller/user-cluster-controller-manager/user-cluster-controller/resources`, but this is not the PR to move a ton of files around. As this piece of code is for a resource that lives inside the usercluster, `resources/resources/ca-bundle` is IMHO the right package.

Reply:
ouch... what a unfortunate naming. OK then 🤷‍♂️